hack/test-cmd.sh

```diff
@@ -169,6 +171,23 @@ export OPENSHIFT_PROFILE="${CLI_PROFILE-}"
 # Begin tests
 #
 
+# create master config as atomic-enterprise just to test it works
+atomic-enterprise start \
+  --write-config=$TEMP_DIR/atomic.local.config \
+  --create-certs=true \
+  --master="${API_SCHEME}://${API_HOST}:${API_PORT}" \
+  --listen="${API_SCHEME}://${API_HOST}:${API_PORT}" \
+  --hostname="${KUBELET_HOST}" \
+  --volume-dir="${VOLUME_DIR}" \
+  --etcd-dir="${ETCD_DATA_DIR}" \
+  --images="${USE_IMAGES}"
+
+# ensure that DisabledFeatures aren't written to config files
+! grep -i '\<disabledFeatures\>' \
+	"${MASTER_CONFIG_DIR}/master-config.yaml" \
+	"$TEMP_DIR/atomic.local.config/master/master-config.yaml" \
+	"${NODE_CONFIG_DIR}/node-config.yaml"
```

Comment:
@stevekuznetsov valid items for `disabledFeatures` array are: `Builder`, `S2IBuilder` and `WebConsole`. See [here](https://github.com/openshift/origin/blob/master/pkg/cmd/server/api/types.go#L15). But they shouldn't be written to configuration files (at least with what I committed). Now it seems they will be written nevertheless, because their [definitio](https://github.com/openshift/origin/blob/master/pkg/cmd/server/api/v1/types.go#L152)  lacks `omitempty`. So I wonder is this working at all?

I tried writing a master config. The entry is there (`disabledFeatures: null`).